packages/material-ui/src/locale/index.js

```diff
@@ -1092,14 +1180,18 @@ export const ptBR = {
 };
 
 export const ptPT = {
+  localization: {
+    formatNumber: formatNumber('pt-PT'),
+  },
   props: {
     // MuiBreadcrumbs: {
     //   expandText: 'Show path',
```

Comment:
@hrafaelveloso Thank you for the initial contribution. We have a couple of new keys, if you have some time, it would be awesome to translate them :).